constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -231,6 +245,72 @@ export class ClassEditPage extends React.Component {
                         onChange={(e) => this.handleChange(e, "inputClassKey")}
                     />
 
+                    <InputText
+                        label="Google Classroom Code"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter the google classroom code"
+                        value={this.state.googleClassCode}
+                        onChangeCallback={(e) =>
+                            this.handleChange(e, "googleClassCode")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
+                    />
+
+                    <InputText
+                        label="Price Lump"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter price for one time payment (Either enter only in this field or only in the price per session field)"
+                        onChangeCallback={(e) =>
+                            this.handleIntegerChange(e, "priceLump")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
+                    />
+
+                    <InputText
+                        label="Price Per Session"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter price for pay per session attended (Either enter only in this field or only in the price lump field)"
+                        onChangeCallback={(e) =>
+                            this.handleIntegerChange(e, "pricePerSession")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
```

Comment:
A class should either have `pricePerSession` filled or `priceLump` filled or neither. If both of these fields are filled, they should be valid numbers (i.e. can't input abc as a price).

Since we're checking for two conditions, `validators` can be an array of two validators. One validator per condition & message.
```
validators={[
    {
         validate: (input) => // check valid uninteger,
         message: "price must be a valid number"
    },
    {
        validate: (input) => // check that if this input is not "", this.state.priceLump should be a valid number
        message: "Only one of PriceLump or PricePerSession should be filled. Both cannot be filled"
    }
]}
```

Then do the same for the priceLump input